packages/tools/devtools/devtools-core/src/IFluidDevtools.ts

```diff
@@ -33,6 +34,15 @@ export interface IFluidDevtools extends IDisposable {
 	 */
 	registerContainerDevtools(props: ContainerDevtoolsProps): void;
 
+	/**
```

Comment:
While we're making changes to this interface, would you mind tagging it as `@sealed`? We don't want external users to be extending this interface.